<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1760</width>
    <height>939</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Source Insight</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCustomPlot" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>350</y>
      <width>731</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>1011</width>
      <height>781</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Abyssinica SIL</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>全局拓扑资源信息</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_text">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>0</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Gothic L</family>
      <pointsize>33</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TSN INSIGHT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>120</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Abyssinica SIL</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TSN流特征信息</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_1">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>160</y>
      <width>421</width>
      <height>121</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string notr="true">源mac</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>目的mac</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>时间周期(us)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>最大延迟(us)</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="flags">
      <set>NoItemFlags</set>
     </property>
    </item>
   </widget>
   <widget class="QTableWidget" name="tableWidget_2">
    <property name="geometry">
     <rect>
      <x>1430</x>
      <y>160</y>
      <width>321</width>
      <height>121</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>源mac</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>目的mac</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>预约带宽(Mb/s)</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1480</x>
      <y>120</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Abyssinica SIL</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>带宽预约流特征信息</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>300</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Abyssinica SIL</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>全局时钟同步状态展示</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_3">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>730</y>
      <width>741</width>
      <height>151</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>TSN队列0</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>TSN队列1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>带宽预约队列</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>BEST EFFORT队列</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>交换机0</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>交换机1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>交换机2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>交换机3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>交换机4</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>交换机5</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>690</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Abyssinica SIL</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>交换机调度信息</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_30">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>360</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>1号</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3_2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>580</y>
      <width>201</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>交换机3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2_1">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>520</y>
      <width>201</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>主机2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3_1">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>730</y>
      <width>181</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>主机3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>350</y>
      <width>171</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>交换机2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_45">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>440</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>2号</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_48">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>500</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>3号</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_1_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>230</y>
      <width>201</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>交换机1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_1_1">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>70</y>
      <width>191</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>主机1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_55">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>510</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>4号</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_0_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>210</y>
      <width>191</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>交换机0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_0_1">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>201</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>主机0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_64">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>430</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>5号</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>320</y>
      <width>171</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>交换机5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>191</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>主机5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_71">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>360</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>0号</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>560</y>
      <width>181</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>交换机4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4_1">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>730</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>主机4</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>29</y>
      <width>1741</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>50</y>
      <width>20</width>
      <height>851</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>50</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Abyssinica SIL</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TSN运行状态展示</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
